app/models/comment.rb

```diff
@@ -0,0 +1,6 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+
+  validates :body, presence: true, length: { in: 200..500 }
```

Comment:
### [OPTIONAL]

- [ ] You can make the range for the length smaller so it is easier to test without having to think of 200 characters.